File: hw/santorum/u3/hw29Singleton/JavaScript/utils/EmailService.js
import EmailConfig from "./EmailConfig.js";

export default class EmailService {
  static async sendEmail(to, message) {
    const config = await EmailConfig.getInstance();

    console.log("----- EMAIL SENT -----");
    console.log("From:", config.senderEmail);
    console.log("To:", to);
    console.log("Message:", message);
    console.log("----------------------");
  }
}

File: hw/santorum/u3/hw29Singleton/Python/utils/email_service.py
from utils.email_config import EmailConfig

class EmailService:

    @staticmethod
    def send_email(to, message):
        sender = EmailConfig().get_sender_email()

        print("----- EMAIL SENT -----")
        print(f"From: {sender}")
        print(f"To: {to}")
        print(f"Message: {message}")
        print("----------------------")
